views/partials/email_partials/email_single_breach.hbs

```diff
@@ -3,9 +3,9 @@
       <td class="email-breach-image" valign="top" style="padding-bottom: 10px; width: 40px;">
         <img class="breach-logo" 
           {{#if breachAlert }}
-            src="{{ SERVER_URL }}/img/logos/missing-logo-icon.png"
+            src="{{ LogoUrl }}"
           {{else}}
-            src="{{ SERVER_URL }}/img/logos/{{ LogoPath }}"
+            src="{{ LogoUrl }}"
```

Comment:
Looks like the `{{#if}}` and `{{else}}` blocks are the same here. Not sure if we can remove the check, or if we need different logic in one of the paths.

Reply:
What he said